### PR TITLE
Removed dependency of lab 11 on lab 9

### PR DESCRIPTION
**Pull request by Rokory**
In this fork, I removed the dependency of lab 11 on lab 9. If time is short, participants can choose to skip lab 9.

### CODE REVIEW

File: Instructions/20347A_LAB_04.md [hunk 1]
```diff
@@ -181,7 +188,9 @@ The main tasks for this exercise are as follows:
 
 5. Close Internet Explorer, open it again, and connect to the Office 365 admin center. Sign in as  **Holly@adatumyyxxxxx.hostdomain.com** using the password **Pa55w.rd**.
 
-6. From the Office 365 admin center, download and install Azure AD Connect with Customized Settings. You will need to configure the security settings for the Internet zone to enable file downloads.
+6. From the Office 365 admin center, download and install Azure AD Connect with Customized Settings.
+
+  >  **Note:** Alternatively you can also search for Microsoft Azure Active Directory Connect in the Microsoft Download Center and download it from there.
```
Comment (itbland) [inline Instructions/20347A_LAB_04.md]: Has Microsoft removed all old versions of AD Connect from the Download site? With DirSync, you never knew if you were getting the latest version (and there may have been a question on the test about ONLY obtaining DirSync from the Office 365 or Azure admin center).

File: Instructions/20347A_LAB_AK_04.md [hunk 2]
```diff
@@ -177,65 +173,67 @@
...
+19. If a  **Windows Internet Explorer 10** dialog box appears, select **Use recommended security and compatibility settings**, and then click  **OK**.
 
-19. Click the  **Admin** tile, and then click **Go to the old admin center**. 
+20. In the  **Address** box, type **https://portal.office.com**, and then press Enter.
 
-20. In the Office365 admin center, in the left side menu, click  **USERS**, and then click  **Active users**.
+21. Sign in as  **holly@Adatumyyxxxxx.hostdomain.com**, using the password  **Pa55w.rd**.
 
-21. To the right of  **Active Directory synchronization**, click  **Manage** (or if Active Directory synchronization has not yet completed, click **Set up**).
+22. In the Office365 admin center, in the left side menu, click  **Health**, and then click  **Directory Synch Status**.
```
Comment (itbland) [inline Instructions/20347A_LAB_AK_04.md]: I agree that this is a much better way to show the students how to check. 
In English the label is: **Directory Sync Status** (without the "h")

File: Instructions/20347A_LAB_AK_11.md [hunk 3]
```diff
@@ -112,17 +112,17 @@
 
 12. Send the message.
 
-13. In Microsoft Edge, connect to https://adatumyyxxxxx.sharepoint.com/sites/marketing.
```
Comment (itbland) [inline Instructions/20347A_LAB_AK_11.md]: Brilliant move! Please approve this!

File: Instructions/20347A_LAB_04.md (hunk 1 above)
Comment (Rokory) [inline Instructions/20347A_LAB_04.md]: I could not find any old version in the download center. Moreover, as I remember from the DirSync era, the version in the download center was newer sometimes. The main purpose of this note is to enable participants to completely skip exercise 1, if time runs short.

**Comment by JasonHershey** [comment]
@itbland and @rokory - regarding the changes related to removing dependencies. Based on comments it sounds like the reasons to make these changes are:
- Remove dependencies so earlier lab exercises can be skipped
- Remove dependencies so students can complete later labs even if they fail to perform tasks in earlier labs

Does that some it up?
Does this imply a problem with the course timing? (is there not enough time allowed?)
Similarly, is this implying there is a problem with the earlier lab tasks.. they are too hard for students to complete?

**Comment by itbland** [comment]
I almost always have my students skip labs 8 & 9 and tell them that they are able to do them AFTER the hosted lab is gone (limit of 120 hrs from start).

I haven’t had time to comment more fully—I will try to get to this this evening.

-Eric

**Comment by Rokory** [comment]
The course is tough in timing, but doable, if you have experienced students. Sometimes students have more questions in the theory lessons, sometimes, they'd like to experiment with some features. So it is always good to have flexibility. Most other Microsoft courses do not have dependent labs. So they can modularized and taught in shortened versions. E. g. there might be students, who are very experienced with SharePoint. The usually choose to skip the lab, because it does not teach them anything new. But they'd like to do the lab on compliance.

I will issue more pull requests, as I also want to resolve the dependencies of later labs on the DirSync lab. There are students, who are not interested in directory synchronization for various reasons. There should be the option to skip this lab.

**Comment by itbland** [comment]
Because this is the ONLY O365 class, I find it rare to actually get only qualified students in class—probably ~40% can’t really handle the labs (or even some of the concepts), so the timing is too tight because we can’t redirect students to anything else easier/more appropriate.
That being said, in Mod 11, using the Team site instead of Marketing—I don’t see an impact, I like that suggestion.
However, In Mod 05, substituting Holly for Beth on CL3 & CL4 will then miss the reinforcement of the message that non-admins can’t install ProPlus, so I’m not so sure that is a good idea. If we try to use a different account that is not a local admin, but has an O365 account, we will have to wait for the profile to be created, again, not a good thing (unless we get the VMs updated).

My $.02
-Eric

**Comment by Rokory** [comment]
I agree with itbland about his criticism on my version of lab 5. So I create a new commit with some more changes.

The goal was to remove as many dependencies on lab 4 as possible. Now all labs only depend on successful completion of lab 2, some also depend on lab 3. There are two notable exceptions:

Lab 12 has a "natural" dependency on lab 4.
In lab 5 there is a small dependency regarding one user, synced in lab 4, because of instructional purposes. If students did not complete lab 4, they are able to skip this one and only step without affecting the rest of the lab.

The trick is to let students create one additional user, named Beth Burke in lab 2A. This user is synced later in lab 4, which is fine, because it might show users how linking existing O365 accounts during sync works. In lab 2B I let them create Abbi Skinner using PowerShell. This user is also linked using directory sync later.

Please commit my pull request. I think, this is a great improvement for the course to simplify delivery.

**Comment by itbland** [comment]
Why don’t we create the Beth account in lab 2A, instead of one of the other accounts that are not used later on? (and then just sync in mod 04 and match the existing pilot user—like we did in the old version)
(In the old lab I had mapped out which accounts got used later so if someone “blew up” their lab, we could more quickly recover in a new lab, but I have not had time to do the same with this new version.)

-Eric

**Comment by JasonHershey** [comment]
Thanks guys for the explanation. I can merge in changes related to errors in the labs and changes to Office 365.  Requests to change the design of the course, like changing lab/module dependencies are  something I can help track, but are 'out of scope' for changes to current version of the course.  If we can updating the pull requests to include only the fixes, I can easily approve them.  Otherwise, it will take me a little time to approve/reject individual changes.    For tracking change requests, what I've seen in other courses like [course 20533](https://github.com/MicrosoftLearning/20533-ImplementingMicrosoftAzureInfrastructureSolutions) is that they have used  **labels** to indicate enhancements or design change requests.  Not sure of schedule, but I suspect this course will get a full update in the fall.

**Comment by Rokory** [comment]
This pull request only contains design changes. I have all fixes in another branch, which I created a separate pull request for (also pending). However, I hope we all agree on minimizing lab dependencies is a good thing. So please consider these changes on future updates to the course.

**Comment by JasonHershey** [comment]
Thanks. Will close this one out but keep it around for course updates.

**Pull request closed without merging**